content/derisking/index.html

```diff
@@ -0,0 +1,95 @@
+---
+title: De-risking government technology
+permalink: /derisking/
+layout: layouts/wide
+tags: derisking
+eleventyNavigation:
+  key: derisking-home-index
+  parent: derisiking-home
+  order: 1
+  title: Derisking Guide
+---
+
+<section class="usa-section--dark hero padding-y-6">
+  <div class="grid-container">
+    <div class="grid-row grid-gap flex-align-center">
+      <div class="tablet:grid-col-6">
+        <h1 class="tablet:font-heading-2xl section__heading--white margin-y-0">Deliver successful technology projects</h1>
+        <p class="font-heading-lg line-height-sans-3">Guidance for delivering successful custom technology projects in government.</p>
+        <a class="usa-button usa-button--accent margin-bottom-105" href="#guide-teasers">Read the guides</a>
+        <a class="usa-button usa-button--outline usa-button--inverse" href="#workshops">Request a workshop</a>
+      </div>
+      <div class="tablet:grid-col-6 display-flex tablet:flex-justify-center margin-top-2">
+        {% image_with_class "assets/derisking/img/guide-covers-circle.png" "maxw-mobile" "guide covers" %}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="usa-graphic-list usa-section usa-prose padding-bottom-0">
+  <div class="grid-container">
+    <div class="usa-graphic-list__row grid-row grid-gap">
+      <div class="usa-media-block">
+        {% image_with_class "assets/derisking/img/pie-chart-186.png" "tagline__image usa-media-block__img" "" %}
+        <div class="usa-media-block__body measure-5">
+          <h2 class="">
+            Only 13% of large government IT projects&nbsp;succeed<sup>*</sup>
+          </h2>
+          <p>Implementing custom software projects can be extraordinarily costly and risky in a government setting. Waterfall software development remains standard at all levels of government. Outdated budgeting and oversight processes have been designed around these very waterfall processes. Moving toward a user-centered, agile approach to this work will save millions of dollars in spending on bad software.</p>
+          <p class="font-body-2xs text-base-dark">
+            <span class="text-primary"> * </span> From The Standish Group’s &ldquo;Haze&rdquo; report
```

Comment:
Ooh is there a link for this?

Reply:
There isn't on the original but this _might_ have been intentional.  I am leery of adding it here because of the approvals this went through to get published, so I suggest we leave it off.